src/components/BannerDescription/BannerDescription.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+
+import { BannerDescription, Props } from './BannerDescription';
+
+export default {
```

Comment:
Based on [CODE_GUIDELINES](https://github.com/chanzuckerberg/edu-design-system/blob/next/docs/CODE_GUIDELINES.md#conventions-for-compound-components), these subcomponents shouldn't have their own story or test files -- rather, it'll be covered by the Parent component

Reply:
Oops, I missed that, thanks!